kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -34,14 +35,23 @@ export class FormWizard extends HTMLElement {
   static get markup() {
     return `
       <template>
-        <div class="form-wizard-content">
-          <h2>${gettext("Wizard header")}</h2>
-          <section>
-            <ul id="step-indicator"></ul>
-            <slot name="active"></slot>
-          </section>
+        <div class="form-wizard-root">
+          <div class="form-wizard-content">
+            <h2>${gettext("Wizard header")}</h2>
+            <section>
+              <ul id="step-indicator"></ul>
+              <slot name="active"></slot>
+            </section>
+          </div>
+          <progress max="100" value="0"></progress>
+          <div class="form-wizard-disqualified">
+            <div class="disqualification">
+              <img class="warning-icon" src="${warningImageURL}"/>
+              <span class="disqualification-header"></span>
```

Comment:
<img width="215" alt="Screenshot 2023-05-18 at 5 41 11 PM" src="https://github.com/mozilla/kitsune/assets/15138046/11d961e7-f28d-45b8-8e82-05c0f1f508cb">


Splitting hairs here a bit, but the icon isn't quite vertically centered with the text. Maybe worth wrapping them in a flex container?